.github/workflows/publish-staff-site.yml

```diff
@@ -0,0 +1,47 @@
+on:
+  workflow_dispatch:
+  push:
+    branches: main
+jobs:
+  publish:
+    name: publish to stat20staff netlify
+    runs-on: ubuntu-latest
+    steps:
+      # for accessing this repo's assets in the container
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      # run the container, attaching this repo as a working directory
+      - name: Run the build process with Docker
+        uses: addnab/docker-run-action@v3
+        with:
+          image: stat20/stat20-docker:latest
+          options: -v ${{ github.workspace }}:/home/rstudio/stat20
+          shell: bash
+          run: |
+            /assets/scripts/propagate-dates.r
+            /assets/scripts/write-toc.r
+            quarto publish netlify --id 33fedc67-c501-4720-bd24-6e00c15aa700
+
```

Comment:
I manually tried this, setting QUARTO_PROFILE=staff-site, without NETLIFY_AUTH_TOKEN (because I don't know it), and after altering some of the .qmd, and quarto ran with no output and no error exit code. It did not seem to actually update the online textbook. This is expected, although I would have expected quarto to complain about the missing netlify token.